syntax = "proto3";
message NodeInfo
{
  uint32 node_id = 1;
  uint32 node_type = 2;
}
